packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -85,6 +87,12 @@ class InteractiveViewer extends StatefulWidget {
            && boundaryMargin.left.isFinite)),
        super(key: key);
 
+  /// If true, panning is only allowed in the direction of the main axes.
```

Comment:
nit: I couldn't find the definition of "main axes". Maybe "horizontal axis or vertical axis"?

Reply:
Will do, that sounds better.